net-p2p/qbittorrent/qbittorrent-4.6.0.ebuild

```diff
@@ -0,0 +1,138 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake edo multibuild systemd xdg
+
+DESCRIPTION="BitTorrent client in C++ and Qt"
+HOMEPAGE="https://www.qbittorrent.org"
+
+if [[ ${PV} == *9999 ]]; then
+	EGIT_REPO_URI="https://github.com/qbittorrent/qBittorrent.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/qbittorrent/qBittorrent/archive/release-${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~riscv ~x86"
+	S="${WORKDIR}"/qBittorrent-release-${PV}
+fi
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="+dbus +gui qt6 test webui systemd"
+RESTRICT="!test? ( test )"
+REQUIRED_USE="|| ( gui webui )"
+
+RDEPEND="
+	>=dev-libs/openssl-1.1.1:=
+	>=net-libs/libtorrent-rasterbar-1.2.18:=
+	>=sys-libs/zlib-1.2.11
+	virtual/libiconv
+	gui? (
+		dev-libs/geoip
+		!qt6? (
+			dev-qt/qtgui:5
+			dev-qt/qtsvg:5
+			dev-qt/qtwidgets:5
+			dbus? ( dev-qt/qtdbus:5 )
+		)
+		qt6? (
+			dev-qt/qtbase:6[dbus?,gui,widgets]
+			dev-qt/qtsvg:6
+		)
+	)
+	qt6? ( dev-qt/qtbase:6[network,ssl,sql,sqlite,xml(+)] )
+	!qt6? (
+		dev-qt/qtcore:5
+		dev-qt/qtnetwork:5[ssl]
+		dev-qt/qtsql:5[sqlite]
+		dev-qt/qtxml:5
+	)
```

Comment:
These should probably be grouped under `gui` conditions, as `-gui +qt6` does nothing right?